var/spack/repos/builtin/packages/openfoam-org/package.py

```diff
@@ -107,8 +109,9 @@ class OpenfoamOrg(Package):
     assets = ['bin/foamEtcFile']
 
     # Version-specific patches
+    patch('50-etc.patch', when='@5.0:')
```

Comment:
Where is that patch coming from?

Reply:
This patches it to very closely resemble what the openfoam-com package has, but now using `pwd -P` (like the org version does) instead of  `pwd -L` like we do. I'll add some comments on the diffs.